src/shared/components/post/post-listing.tsx

```diff
@@ -748,6 +768,7 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
         title={title}
         to={`/post/${post_view.post.id}?scrollToComments=true`}
         data-tippy-content={title}
+        target={this.linkTarget}
```

Comment:
Is there a way to add [rel="noopener"](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noopener)?

Reply:
it's implicit on target="_blank"